config/secrets_config.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'invoca_secrets'
+
+# Note: normally we'd put this in config/initializers, but the web repo needs to have secrets configured very early
+# so that ActiveTableSet can be configured very early to support the bootstrap process. Both are configured from
+# application.rb.
```

Comment:
don't think these 3 comment lines apply here..  and to that end, should this move to `config/initializers` ?